src/executables/utility.h

```diff
@@ -738,6 +739,17 @@ typedef enum
   FLASHBACK_MSG_USAGE = 60
 } MSGCAT_FLASHBACK_MSG;
 
+/* Message id in the set MSGCAT_UTIL_SET_MEMMON */
+typedef enum
+{
+  MEMMON_MSG_EXCLUSIVE_OPTION = 1,
+  MEMMON_MSG_NOT_SUPPORTED = 2,
+  MEMMON_MSG_NO_MATCHING_MODULE = 3,
+  MEMMON_MSG_INVALID_TRAN_COUNT_OPTION = 4,
+  MEMMON_MSG_NOT_IN_STANDALONE = 59,
```

Comment:
```suggestion
  MEMMON_MSG_NOT_IN_STANDALONE = 5,
```
Why is this value specifically 59?

Reply:
I think this is a legacy because it is a general error, so there is an intention to reserve blank space for future utility-specific errors.